### DISCUSSION TITLE
GPy users mailing list

### DISCUSSION THREAD

**Discussion by Atulsingh92** [SheffieldML/GPy#934] [Q&A]
Hello, 
I wanted to register with the user mailing list mentioned on [this](https://sheffieldml.github.io/GPy/) page.

But the [link](https://lists.shef.ac.uk/sympa/subscribe/gpy-users) doesn't seem to open. Could someone please update this?

**Comment by lawrennd** [comment]
Thanks for mentioning @Atulsingh92, that page is quite out of date. @ekalosak, I can't see how that propagates from a release update ... presumably some step that we're not aware of? Or something I need to intervene on as an organisation owner?
